content/en/v3/develop/reference/jx/admin/operator/_index.md

```diff
@@ -58,7 +58,7 @@ Installs the git operator in a cluster
       --no-log                      to disable viewing the logs of the boot Job pods
       --no-switch-namespace         to disable switching to the installation namespace after installing the operator
       --set stringArray             one or more helm set arguments to pass through the git operator chart. Equivalent to running 'helm install --set some.name=value'
-      --setup stringArray           a git configuration command to configure git inside the git operator pod to deal with things like insecure docker registries etc. e.g. supply 'git config --global http.sslverify false' to disable TLS verification
+      --setup stringArray           a git configuration command to configure git inside the git operator pod to deal with things like insecure docker registries etc. e.g. supply 'git config --global http.sslVerify false' to disable TLS verification
```

Comment:
@mheuser , this is autogenerated from jx-admin repo: https://github.com/jenkins-x-plugins/jx-admin/blob/a300172dad9c964ab1ac0a30fff2d15358a515de/pkg/cmd/operator/operator.go#L107.
So you need to fix it there, and run `make docs` to regenerate the docs for the changes to persist.
I am going to merge this here, but it would be nice if you also open a PR in jx admin repo.